lib/pure/asyncdispatch.nim

```diff
@@ -1580,8 +1580,8 @@ proc createAsyncNativeSocket*(domain: Domain = Domain.AF_INET,
                            protocol: Protocol = IPPROTO_TCP): AsyncFD =
   createAsyncNativeSocketImpl(domain, sockType, protocol)
 
-proc newAsyncNativeSocket*(domain: cint, sockType: cint,
-                           protocol: cint): AsyncFD {.deprecated: "use createAsyncNativeSocket instead".} =
+proc newAsyncNativeSocket*(domain: cint, sockType: cint, protocol: cint):
+    AsyncFD {.deprecated: "use createAsyncNativeSocket instead".} =
```

Comment:
I also prefer the existing format here.

Reply:
> I also prefer the existing format here.

The problem is that the existing format has too long line, and nimpretty is quite strict about line lengths. (Too strict sometimes, if you ask me)